src/api/routes/summaries.ts

```diff
@@ -180,3 +190,48 @@ export async function findMissingCrudeSummaries(): Promise<CrudeSummaryDescripto
   }
   return ret;
 }
+
+export async function extractHandlebars(transcriptId: string): Promise<string[]> {
+  const summaries = await db.Summary.findAll({ where: { transcriptId } });
+
+  // find all handlebar from summaries under one transcript
+  let handlebars: string[] = [];
+  for (let s of summaries) {
+    const matches = s.summary.match(/{{(.*?)}}/g);
+    if (matches) {
+      // Extract matched handlebars and append to the existing handlebars
+      const extractedHandlebars = matches.map(match => match.slice(2, -2).trim());
```

Comment:
why do you need trim here?

Reply:
I have trim here to get rid of the curly brackets `{{ }}` wrapped around the handlebars